transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -268,6 +276,8 @@ void bindInternal(final ResourceAddress address, IoHandler handler,
         //
         final ResourceAddress transportAddress = address.getTransport();
 
+        handshakeTimeoutByTransportAddress.put(transportAddress, address.getOption(HANDSHAKE_TIMEOUT).longValue());
```

Comment:
Check the requirements doc, But I believe it takes the lowest user defined value.  Meaning if the user defines a value higher then the default then you will to the user defined value.  

I would suggest, not defaulting the value in the resource address, but rather keeping it null.  Then at conflict resolution time you will know whether the user set it... : -)